src/components/NanoContract/SelectAddressModal.js

```diff
@@ -35,41 +35,33 @@ import { selectAddressAddressesRequest } from '../../actions';
  * Callback function called when an address is selected
  * @param {() => {}} props.onDismiss
  * Callback function called to dismiss the modal
+ * @param {(item: Object) => {}} props.onEditAddress
+ * Callback function called when an address item is selected for editing
```

Comment:
chore: We have to update these docstrings. Some params were removed and others added